### ISSUE TITLE
"Unknown error code '125'" during file conversion on Windows

### ISSUE THREAD

**Issue by apyrgio** [freedomofpress/dangerzone#630]
A Windows user has reported that they got the following error message in the Dangerzone UI, while converting a file:

```
Unknown error code '125'
```

Most likely, Docker Desktop returned this error code due to a configuration issue. A quick search shows that probably file sharing restrictions are in play here:

* https://forums.docker.com/t/filesharing-not-enabled-volume-sharing-is-not-enabled-on-the-settings-screen-in-docker-desktop-error-125/95332

A temporary solution seems to be to go to the "File Sharing" tab of Docker Desktop, and add the path of the files, so that they can be mounted. A better solution is to fix #443, which is long overdue.

**Comment by apyrgio** [comment]
Turns out that the "File Sharing" tab is only available in case of Hyper-V backends. In WSL2, which is the default in the latest Windows and Docker Desktop versions, all files are shared, so this tab does not exist.

We tried to debug this further via `dangerzone-cli`, but unfortunately we were seeing some different errors, which probably had to do with the command prompt. While experimenting, we did the following:

* Uninstalling and re-installing Docker Desktop,
* Running `wsl --update` as an administrator,
* Copying the file from its original location (an untrusted USB drive) to a local folder.

One of those things did the trick, because afterwards we could convert the file. Unfortunately, we don't know what exactly.
